src/Tizen.NUI/src/public/Events/GestureOptions.cs

```diff
@@ -0,0 +1,331 @@
+        }
+
+        /// <summary>
+        /// Sets the minimum touch events required before a pinch can be started
+        /// </summary>
+        /// <remarks>This is a global configuration option. Affects all gestures.</remarks>
+        /// <param name="number">The number of touch events</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetPinchGestureMinimumTouchEvents(uint number)
+        {
+            Interop.GestureOptions.SetPinchGestureMinimumTouchEvents(number);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Sets the minimum touch events required after a pinch started
+        /// </summary>
+        /// <remarks>This is a global configuration option. Affects all gestures.</remarks>
+        /// <param name="number">The number of touch events</param>
+        public void SetPinchGestureMinimumTouchEventsAfterStart(uint number)
+        {
+            Interop.GestureOptions.SetPinchGestureMinimumTouchEventsAfterStart(number);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Sets the minimum touch events required before a rotation can be started
+        /// </summary>
+        /// <remarks>This is a global configuration option. Affects all gestures.</remarks>
+        /// <param name="number">The number of touch events</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetRotationGestureMinimumTouchEvents(uint number)
+        {
+            Interop.GestureOptions.SetRotationGestureMinimumTouchEvents(number);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Sets the minimum touch events required after a rotation started
+        /// </summary>
+        /// <remarks>This is a global configuration option. Affects all gestures.</remarks>
+        /// <param name="number">The number of touch events</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetRotationGestureMinimumTouchEventsAfterStart(uint number)
+        {
+            Interop.GestureOptions.SetRotationGestureMinimumTouchEventsAfterStart(number);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Sets the minimum holding time required to be recognized as a long press gesture
+        /// </summary>
+        /// <remarks>This is a global configuration option. Affects all gestures.</remarks>
+        /// <param name="time">The time value in milliseconds</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SetLongPressMinimumHoldingTime(uint time)
```

Comment:
same as above.
and as you know, if a property is used for a simple interface, user can get the saved value easily.